gtfs_realtime_translators/registry/registry.py

```diff
@@ -12,16 +13,17 @@ class TranslatorKeyWarning(Warning):
 
 class TranslatorRegistry:
     TRANSLATORS = {
-        'la-metro': LaMetroGtfsRealtimeTranslator,
+        'la-metro-old': LaMetroGtfsRealtimeTranslator,
```

Comment:
Should we add an entry for `la-metro` that points to SwiftlyGtfsRealtimeTranslator? Or just repoint feeds using `swiftly` translator key?

Reply:
Actually, nvm. I think we should prob just use `swiftly` for any swiftly feed. You'll notice `septa` is using Swiftly right now (thats the only xltr available). We can likely repoint those too.

Reply:
Yeah I think the plan should be keep what we have, and everything else that was using Swiftly should explicitly switch over to using that name